README.md

```diff
@@ -66,18 +68,18 @@ Once you're done with the above setup (no matter which method you used), run `np
 
```

Comment:
At the top where the `## Initial Setup` is, we should add a note that if you are running from the vagrant ssh, these are all installed (node, composer and wp-cli)

Reply:
Also for the manual part, besides running `npm start`, shouldn't `composer install`be ran as well? Line 41

Reply:
You still need `wp-cli` for vagrant (since `wp core download` is run outside vagrant). And you don't actually need composer regardless (at least not for installation) since we're running it using `npx`.

As for composer install, that is done just after renaming (actually it's `update` not `install`). (1. `npm install`, 2. renaming, 3. `npx composer update`, 4. `wp core donwload`)